src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -201,16 +230,7 @@ export const QueryResultPane = () => {
     ) => {
         const divId = `grid-parent-${batchId}-${resultId}`;
         return (
-            <div
-                id={divId}
-                className={classes.queryResultContainer}
-                style={{
-                    height:
-                        totalResultCount === 1
-                            ? "100%"
-                            : (100 / totalResultCount).toString() + "%",
```

Comment:
You can remove totalResultCount parameter since it's no longer used.
<img width="905" alt="image" src="https://github.com/user-attachments/assets/5b6b00c3-604d-4c3d-86b8-5b5642d0e046">